src/util/state-icon.js

```diff
@@ -13,6 +13,12 @@ function binarySensorIcon(state) {
       return activated ? 'mdi:water-off' : 'mdi:water';
     case 'safety':
     case 'gas':
+    case 'light':
+      return activated ? 'mdi:brightness-5' : 'mdi:brightness-7';
+    case 'sound':
+      return activated ? 'mdi:bell-off' : 'mdi:bell-ring';
```

Comment:
We wouldn't want to overlap with the alarm component. What about `music-note` and `music-note-off`  ?